src/pybind/mgr/dashboard/module.py

```diff
@@ -405,6 +407,26 @@ def set_mgr_created_self_signed_cert(self):
         if result.retval != 0:
             return result
         return 0, 'Self-signed certificate created', ''
+    
+    @CLICommand("dashboard feedback get")
+    def get_issues_cli(self, issue_number: str):
+        cephTrackerClient = CephTrackerClient()
+        response = cephTrackerClient.get_issues(issue_number)
```

Comment:
I suppose the commits regarding the backend will get deleted after the [backend pr](https://github.com/ceph/ceph/pull/42090) is merged?

Reply:
yes